src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -651,14 +666,25 @@ namespace
 
         void create_bounce_settings(QFormLayout* layout, const string& prefix)
         {
-            const string widget_base_key = prefix + ".bounces.";
 
+            const string widget_base_key = prefix + ".bounces.";
             QSpinBox* max_bounces = create_integer_input(widget_base_key + "max_bounces", 0, 10000, 1);
             QCheckBox* unlimited_bounces = create_checkbox(widget_base_key + "unlimited_bounces", "Unlimited");
+            
+            if(prefix == "photon_tracing")
```

Comment:
Insert a space after `if` (and all such keywords: `for`, `while`, `do`, `else`, etc.)